### PR TITLE
[13.0] [MIG] currency_rate_update

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

File: currency_rate_update/__manifest__.py [hunk 1]
```diff
@@ -3,29 +3,22 @@
...
-    'license': 'AGPL-3',
-    'category': 'Financial Management/Configuration',
-    'summary': 'Update exchange rates using OCA modules',
-    'depends': [
-        'base',
-        'mail',
-        'account',
+    "name": "Currency Rate Update",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp, " "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (alexey-pelykh) [inline currency_rate_update/__manifest__.py]: The `" "` parts look really odd

File: currency_rate_update/models/res_currency_rate_provider.py [hunk 2]
```diff
@@ -156,157 +134,132 @@ def _update(self, date_from, date_to, newest_only=False):
...
                 raise UserError(
-                    _(
-                        'Invalid rate from %(provider)s for'
-                        ' %(currency)s : %(rate)s'
-                    ) % {
-                        'provider': self.name,
-                        'currency': currency.name,
-                        'rate': rate,
-                    }
+                    _("Invalid rate from %(provider)s for" " %(currency)s : %(rate)s")
```
Comment (alexey-pelykh) [inline currency_rate_update/models/res_currency_rate_provider.py]: `" "`again, seems like an autoformatter

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
Overall looks good, yet some `" "` in one line look not great

**Comment by feketemihai** [comment]
@alexey-pelykh I updated the formatting, since it's coming from pre-commit.

File: currency_rate_update/models/res_currency_rate_provider.py (hunk 2 above)
Comment (feketemihai) [reply currency_rate_update/models/res_currency_rate_provider.py]: updated

File: currency_rate_update/__manifest__.py (hunk 1 above)
Comment (feketemihai) [reply currency_rate_update/__manifest__.py]: updated

Comment (alexey-pelykh) [reply currency_rate_update/__manifest__.py]: hm, does not show :(

File: currency_rate_update/models/res_currency_rate_provider_ECB.py [hunk 3]
```diff
@@ -3,62 +3,95 @@
...
-    _inherit = 'res.currency.rate.provider'
+    _inherit = "res.currency.rate.provider"
 
-    service = fields.Selection(
-        selection_add=[('ECB', 'European Central Bank')],
-    )
+    service = fields.Selection(selection_add=[("ECB", "European Central Bank")])
 
-    @api.multi
+    @api.model
```
Comment (alexey-pelykh) [inline currency_rate_update/models/res_currency_rate_provider_ECB.py]: Why?

Comment (alexey-pelykh) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]: It supposed to be run on instance, not on class

Comment (feketemihai) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]: @alexey-pelykh Right, reverted

File: currency_rate_update/__manifest__.py (hunk 1 above)
Comment (alexey-pelykh) [reply currency_rate_update/__manifest__.py]: Seems autoformatting `" "` is still there

File: currency_rate_update/models/res_currency_rate_provider.py (hunk 2 above)
Comment (alexey-pelykh) [reply currency_rate_update/models/res_currency_rate_provider.py]: Odd, but I don't see the update :(

**Comment by alexey-pelykh** [comment]
@feketemihai https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch this suggests a bit different approach regarding that auto-formatting should've been a separate commit

**Comment by MiquelRForgeFlow** [comment]
@feketemihai yeah, please, split that commit into two, one for new format and the other the proper migration. If not, it's very difficult to review.

**Comment by MiquelRForgeFlow** [comment]
and please cherry-pick https://github.com/OCA/currency/pull/71 :)

**Comment by feketemihai** [comment]
@MiquelRForgeFlow @alexey-pelykh updated according with comments. Thanks

**Review by alexey-pelykh — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**